logging/bin/remove_logging_open.sh

```diff
@@ -50,7 +50,13 @@ if [ "$LOG_DELETE_CONFIGMAPS_ON_REMOVE" == "true" ]; then
   kubectl delete configmap --ignore-not-found -n $LOG_NS -l managed-by=v4m-es-script
 fi
 
-removeV4MInfo "$LOG_NS"
+# If a deployment with the old name exists, remove it first
+if helm3ReleaseExists "v4m" $LOG_NS; then
+  removeV4MInfo "$LOG_NS" "v4m"
+else
```

Comment:
I don't think we want the removal of the "v4m-logs" to be conditional here.  Regardless of whether was (or was not) a "v4m" release, we want to remove "v4m-logs".

How does the removeV4MInfo function handle things if the specified release does NOT exist?  If it handles that without problems (e.g. failing w/o ERRORs) do we need to check if the "v4m" release exists on line 54.  If it does NOT handle that situation well, do we need to add a check that the "v4m-logs" release exists before we attempt to delete it?

Reply:
I was originally worried about there being multiple uninstall error messages like the ones below, but mine doesn't attempt it unless it finds an instance first.  I removed the conditional logic and it should show up in my next push:

> Removing logging components from the [test] namespace [Mon Mar 21 15:59:29 EDT 2022]
>
> INFO Removing Fluent Bit components [v4m-fb] from the [test] namespace [Mon Mar 21 15:59:30 EDT 2022]
> **Error: uninstall: Release not loaded: v4m-fb: release: not found**
> 
> INFO Removing Elasticsearch Exporter [Mon Mar 21 15:59:31 EDT 2022]
> **Error: uninstall: Release not loaded: es-exporter: release: not found**
> 
> INFO Removing Open Distro for Elasticsearch [Mon Mar 21 15:59:31 EDT 2022]
> **Error: uninstall: Release not loaded: odfe: release: not found**